### PR TITLE
No alerting for certificates in error state

### PR DESCRIPTION
**Pull request by MartinWeindel**
**How to categorize this PR?**

/area monitoring
/kind bug

**What this PR does / why we need it**:
If a cluster contains misconfigured certificates, the metrics `cert_management_cert_object_expire` reports a value of 0 and the alerting is triggered as the certificate seems to have expired since about 19600 days.

To exclude such false alerts, the expression has been adapted. 

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```bugfix operator
No alerting for certificates in error state
```

### CODE REVIEW

**Review by ScheererJ — APPROVED** [review]
/lgtm

**Review by istvanballok — COMMENTED** [review]
/hold for a second

File: charts/internal/shoot-cert-management-seed/templates/configmap-observability.yaml [hunk 1]
```diff
@@ -37,7 +37,7 @@ data:
       - name: cert-controller-manager.rules
         rules:
         - alert: SslCertificateWillExpireSoon
-          expr:  (cert_management_cert_object_expire-time())/86400 <= {{ .Values.configuration.certExpirationAlertDays }}
+          expr: cert_management_cert_object_expire > 0 and (cert_management_cert_object_expire-time())/86400 <= {{ .Values.configuration.certExpirationAlertDays }}
```
[the remark this answers is not in the document]
Comment (MartinWeindel) [reply charts/internal/shoot-cert-management-seed/templates/configmap-observability.yaml:40]: Thanks for all the insights and the improved expression.

**Review by istvanballok — APPROVED** [review]
/lgtm

**Review by istvanballok — COMMENTED** [review]
/unhold

**Review by timuthy — APPROVED** [review]
/lgtm

**Pull request merged**